JuiceMaker/JuiceMaker/Model/JuiceMaker.swift

```diff
@@ -1,12 +1,66 @@
-//
-//  JuiceMaker - JuiceMaker.swift
-//  Created by yagom. 
-//  Copyright © yagom academy. All rights reserved.
-// 
-
 import Foundation
 
-// 쥬스 메이커 타입 
 struct JuiceMaker {
     
+    var strawberry = Strawberry()
+    var banana = Banana()
+    var kiwi = Kiwi()
+    var pineapple = Pineapple()
+    var mango = Mango()
+    
+    enum JuiceMakerError: Error {
+        case countUnderZero
+    }
+    
+    func strawberryJuice() throws {
```

Comment:
보통 메서드명이 명사형인경우에는 side-effect가 없고 어떤 객체를 return 해주는 경우입니다.
하지만 이 메서드는 과일의 재고에 영향을 주고있네요! 동사형으로 쓰면 적절할 것 같아요
```suggestion
    func makeStrawberryJuice() throws {
```

> Those without side-effects should read as noun phrases, e.g. x.distance(to: y), i.successor().
Those with side-effects should read as imperative verb phrases, e.g., print(x), x.sort(), x.append(y).

https://swift.org/documentation/api-design-guidelines/#naming